tests/runfiles/linux.run

```diff
@@ -355,7 +355,7 @@ tests = ['compress_001_pos', 'compress_002_pos', 'compress_003_pos',
     'compress_004_pos']
 
 [tests/functional/ctime]
-tests = ['ctime_001_pos' ]
+tests = ['ctime' ]
```

Comment:
I think it would be better to switch the two file name, so we get more consistent with others.

Reply:
@tuxoko The reason I didnt do this way was because ctime_001_pos is a .c file and renaming it will mean changing multiple places in the Makefile. If that's ok, I will go ahead and make the change.

Reply:
That's fine, please go ahead and rename it.